plugins/apps/threatbus_misp/README.md

```diff
@@ -9,32 +9,55 @@ Threat Bus MISP Plugin
 
 </h4>
 
-A Threat Bus plugin that enables communication to [MISP](https://www.misp-project.org/).
+A Threat Bus plugin that enables communication with [MISP](https://www.misp-project.org/).
 
 The plugin goes against the pub/sub architecture of Threat Bus (for now),
-because the plugin subscribes a listener to ZeroMQ / Kafka, rather than having
-MISP subscribe itself to Threat Bus. That will be addressed with a MISP module
-in the near future.
+because it actively binds to a single MISP instance to receive attribute
+(IoC) updates from it and to report back sightings via the REST API. Following
+the strict pub/sub architecture of Threat Bus, it *should be the other way
+round*, with MISP binding to Threat Bus. This will eventually be resolved by a
+MISP module.
```

Comment:
This sentence is a bit weird. Perhaps this is better readable:

because it actively binds to a single MISP instance to receive attribute
(IoC) updates, and report back sightings via the REST API.

"other way around" -> american english
"other way round" -> british english 
acc. to this guy: https://www.quora.com/What-is-the-difference-between-other-way-around-and-other-way-round

Reply:
applied your suggestion :+1: